client/gutenberg/extensions/simple-payments/edit.js

```diff
@@ -452,7 +452,7 @@ class SimplePaymentsEdit extends Component {
 	}
 }
 
-const applyWithSelect = withSelect( ( select, props ) => {
+const mapSelectToProps = withSelect( ( select, props ) => {
```

Comment:
Convention is `mapSelectToProps`: https://github.com/WordPress/gutenberg/search?q=mapselecttoprops&unscoped_q=mapselecttoprops